homeassistant/components/nederlandse_spoorwegen/sensor.py

```diff
@@ -47,13 +47,22 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the departure sensor."""
 
     nsapi = ns_api.NSAPI(config[CONF_API_KEY])
+
     try:
         stations = nsapi.get_stations()
     except (
         requests.exceptions.ConnectionError,
         requests.exceptions.HTTPError,
     ) as error:
-        _LOGGER.error("Couldn't fetch stations, API key correct?: %s", error)
+        _LOGGER.error("NS integration could not connect to the internet: %s", error)
+        return
+    except Exception as error:
+        _LOGGER.error("NS integration crashed, please check configuration: %s", error)
+        hass.components.persistent_notification.create(
```

Comment:
We don't want persistent notifications for this case.

Reply:
You would prefer no notification at all, so logs only?

Reply:
Yes, please. There's already a notification if an integration fails to set up.

Reply:
Fixed.